### DISCUSSION TITLE
Add a custom subdomains using name.com

### DISCUSSION THREAD

**Discussion by jusebin** [vercel/community#2750] [Help]
### Summary

Hello there ! 
I bought a domain juru.rocks on name.com
I would like to update the domain of a project and call it tracksinsights.juru.rocks

I did it with a CNAME redirection, but when I tried to add another subdomain throught CNAME for another Vercel project, the first domain failed (and also, the second).

My question is : Can I do some ANAME redirection with vercel ? And if it, how can I do it using name.com ? Thank you in advance

### Steps to Reproduce

a

**Comment by smaeda-ks** [comment]
@jusebin what's the "another subdomain" here?

```
$ dig CNAME tracksinsights.juru.rocks +noall +ans
tracksinsights.juru.rocks. 300	IN	CNAME	cname.vercel-dns.com.
```

I see `tracksinsights.juru.rocks` is set up correctly
